### PR TITLE
Remove PHcalParameters and provide new GT for all scenarios.

### PR DESCRIPTION
**Pull request by diguida**
# Summary of code changes

Remove `PHcalParameters` class, which is replaced by `HcalParameters`. Originally in #12817 
# Summary of GT Changes
## RunI Simulation
- **Ideal scenario**: [`80X_mcRun1_design_v0`](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/80X_mcRun1_design_v0): As [`76X_mcRun1_design_v11`](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/76X_mcRun1_design_v11) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/76X_mcRun1_design_v11/80X_mcRun1_design_v0):
  - removed `PHcalParametersRcd`.
- **Realistic scenario**: [`80X_mcRun1_realistic_v0`](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/80X_mcRun1_realistic_v0): As [`76X_mcRun1_realistic_v11`](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/76X_mcRun1_realistic_v11) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/76X_mcRun1_realistic_v11/80X_mcRun1_realistic_v0):
  - removed `PHcalParametersRcd`.
- **Heavy Ion scenario**: [`80X_mcRun1_HeavyIon_v0`](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/80X_mcRun1_HeavyIon_v0): As [`76X_mcRun1_HeavyIon_v11`](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/76X_mcRun1_HeavyIon_v11) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/76X_mcRun1_HeavyIon_v11/80X_mcRun1_HeavyIon_v0):
  - removed `PHcalParametersRcd`.
- **Proton-Lead scenario**: [`80X_mcRun1_pA_v0`](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/80X_mcRun1_pA_v0): As [`76X_mcRun1_pA_v11`](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/76X_mcRun1_pA_v11) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/76X_mcRun1_pA_v11/80X_mcRun1_pA_v0):
  - removed `PHcalParametersRcd`.
## RunII Simulation
- **Ideal scenario**: [`80X_mcRun2_design_v0`](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/80X_mcRun2_design_v0): As [`76X_mcRun2_design_v14`](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/76X_mcRun2_design_v14) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/76X_mcRun2_design_v14/80X_mcRun2_design_v0):
  - added new Hcal record (`HcalQIETypesRcd`) providing a per-HCAL-readout flag of the FE QIE electronics type
  - removed `PHcalParametersRcd`.
- **Startup scenario**: [`80X_mcRun2_startup_v0`](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/80X_mcRun2_startup_v0): As [`76X_mcRun2_startup_v13`](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/76X_mcRun2_startup_v13) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/76X_mcRun2_startup_v13/80X_mcRun2_startup_v0):
  - added new Hcal record (`HcalQIETypesRcd`) providing a per-HCAL-readout flag of the FE QIE electronics type
  - removed `PHcalParametersRcd`.
- **Asymptotic scenario**: [`80X_mcRun2_asymptotic_v0`](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/80X_mcRun2_asymptotic_v0): As [`76X_mcRun2_asymptotic_v14`](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/76X_mcRun2_asymptotic_v14) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/76X_mcRun2_asymptotic_v14/80X_mcRun2_asymptotic_v0):
  - added new Hcal record (`HcalQIETypesRcd`) providing a per-HCAL-readout flag of the FE QIE electronics type
  - removed `PHcalParametersRcd`.
- **Heavy Ion scenario**: [`80X_mcRun2_HeavyIon_v0`](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/80X_mcRun2_HeavyIon_v0): As [`76X_mcRun2_HeavyIon_v14`](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/76X_mcRun2_HeavyIon_v14) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/76X_mcRun2_HeavyIon_v14/80X_mcRun2_HeavyIon_v0):
  - added new Hcal record (`HcalQIETypesRcd`) providing a per-HCAL-readout flag of the FE QIE electronics type
  - removed `PHcalParametersRcd`.
## RunII Data
- **HLT processing**: [`80X_dataRun2_HLT_frozen_v0`](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/80X_dataRun2_HLT_frozen_v0): As [`76X_dataRun2_HLT_frozen_v11`](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/76X_dataRun2_HLT_frozen_v11) (snapshot time set to 2015-12-18 17:00:00 UTC) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/76X_dataRun2_HLT_frozen_v11/80X_dataRun2_HLT_frozen_v0):
  - added new Hcal record (`HcalQIETypesRcd`) providing a per-HCAL-readout flag of the FE QIE electronics type.
- **HLT Heavy Ion processing**: [`80X_dataRun2_HLTHI_frozen_v0`](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/80X_dataRun2_HLTHI_frozen_v0): As [`76X_dataRun2_HLTHI_frozen_v2`](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/76X_dataRun2_HLTHI_frozen_v2) (snapshot time set to 2015-12-18 17:00:00 UTC) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/76X_dataRun2_HLTHI_frozen_v2/80X_dataRun2_HLTHI_frozen_v0):
  - added new Hcal record (`HcalQIETypesRcd`) providing a per-HCAL-readout flag of the FE QIE electronics type
- **Offline processing**: [`80X_dataRun2_v1`](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/80X_dataRun2_v1): As [`80X_dataRun2_v0`](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/80X_dataRun2_v0) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/80X_dataRun2_v0/80X_dataRun2_v1):
  - added new Hcal record (`HcalQIETypesRcd`) providing a per-HCAL-readout flag of the FE QIE electronics type
  - removed `PHcalParametersRcd`
  - updated B tag discriminators for HLT with the following taggers: `CombinedSVIVFV2RecoVertex`, `CombinedSVIVFV2PseudoVertex`, `CombinedSVIVFV2NoVertex`, `CombinedSVRecoVertex`, `CombinedSVPseudoVertex`, `CombinedSVNoVertex` (as used by HLT data collection in 2015).
## Upgrade 2017 Simulation
- **Ideal scenario**: [`80X_upgrade2017_design_v0`](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/80X_upgrade2017_design_v0): As [`76X_upgrade2017_design_v9`](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/76X_upgrade2017_design_v9) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/76X_upgrade2017_design_v9/80X_upgrade2017_design_v0):
  - added new Hcal record (`HcalQIETypesRcd`) providing a per-HCAL-readout flag of the FE QIE electronics type
  - removed `PHcalParametersRcd`.

### CODE REVIEW

**Comment by diguida** [comment]
please test

File: Configuration/AlCa/python/autoCond.py [hunk 1]
```diff
@@ -2,33 +2,33 @@
...
+    'run1_data'         :   '80X_dataRun2_v1',
     # GlobalTag for Run2 data reprocessing
-    'run2_data'         :   '80X_dataRun2_v0',
+    'run2_data'         :   '80X_dataRun2_v1',
     # GlobalTag for Run1 HLT: it points to the online GT
-    'run1_hlt'          :   '76X_dataRun1_HLT_frozen_v10',
+    'run1_hlt'          :   '80X_dataRun2_HLT_frozen_v0',
     # GlobalTag for Run2 HLT: it points to the online GT
-    'run2_hlt'          :   '76X_dataRun2_HLT_frozen_v10',
+    'run2_hlt'          :   '76X_dataRun2_HLT_frozen_v0',
```
Comment (mmusich) [inline Configuration/AlCa/python/autoCond.py]: @diguida I guess you meant here `80X_dataRun2_HLT_frozen_v0` instead.... the diff looks good [CondDB browers](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/80X_dataRun2_HLT_frozen_v0/76X_dataRun2_HLT_frozen_v11)

Comment (diguida) [inline Configuration/AlCa/python/autoCond.py]: Yeap :-( A typo is always bad: fixing :-)

**Comment by diguida** [comment]
please test

**Comment by diguida** [comment]
+1

**Comment by ianna** [comment]
+1

**Pull request merged**